src/TagsInput/TagsInput.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import type { ViewLayoutEvent } from 'react-native/Libraries/Components/View/ViewPropTypes';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import IconButton from './components/IconButton';
+import TagsContainer from './components/TagsContainer';
+import InputField from './components/InputField';
+import { StyleSheet } from '../PlatformStyleSheet';
+import { Text } from '../Text';
+
+import { MIN_WIDTH } from './constants';
+
+type Props = {|
+  +label: string,
+  +onChangeText: (value: string) => void,
+  +placeholder: string,
+  +selected?: string[],
+  +disabled?: boolean,
+  +fontSize: number,
+  +onClickDelete?: () => void,
```

Comment:
Maybe just unify this with button name to something like `onClear` or `onClearPress`.

Reply:
i prefer onClearPress